advanced/.eslintrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "react-app",
```

Comment:
When I run `yarn run lint`, I get an error that `eslint-config-react-app` can't be found. Maybe this is missing from the `package.json`? Do you get the same thing?

Reply:
well i don't get any error. Just ran 

![screen shot 2018-02-24 at 2 58 41 pm](https://user-images.githubusercontent.com/7845921/36628658-4dc671ea-1973-11e8-9ebd-dbe1617200e7.png)

Let me know you still face error

Reply:
Also check this out https://github.com/facebook/create-react-app/issues/3633

Reply:
@DevanB `create-react-app` uses `react-scripts` internally. And `react-scripts` installs `eslint-create-react-app` so, that won't be visible in the `package.json`.

Reply:
Please follow the following documentation if you want to use `eslint-create-react-app` linter config in your application: https://github.com/facebook/create-react-app/tree/master/packages/eslint-config-react-app

Reply:
Fixed :) with https://github.com/facebook/create-react-app/tree/master/packages/eslint-config-react-app